site/content/guides/using-pw-test-command.md

```diff
@@ -0,0 +1,162 @@
+
+### Create monitors
+
+Add `--create-check` to convert suites of tests into scheduled playwright check suites:
+
+```bash
+# Basic check suite pulling all tests in Chromium every 10 minutes
+npx checkly pw-test --create-check --frequency=10m -- --project=chromium
+
+# Multiple locations pulling all tests tagged critical, and run every 5 minutes
+npx checkly pw-test --create-check --frequency=5m --location=us-east-1 --location=eu-west-1 -- --grep="@critical"
+
+# Mobile tagged tests every 15 mins, in Mobile Chrome and the default location.
+npx checkly pw-test --create-check --frequency=15m -- --project="Mobile Chrome" --grep="@mobile"
+```
+
+## Advanced examples
+
+### Browser and device selection
+
+```bash
+# Desktop browsers
+npx checkly pw-test -- --project="Desktop Chrome" --project="Desktop Firefox"
+
+# Mobile devices
+npx checkly pw-test -- --project="Mobile Safari" --project="Iphone 14"
+```
+
+### Test filtering
+
+```bash
+# All tests tagged smoke
+npx checkly pw-test -- --grep="@smoke"
+
+# Exclude flaky tagged tests
+npx checkly pw-test -- --grep-invert="@flaky"
+
+# Create a check with tests tagged critical that runs every 10 minutes
+npx checkly pw-test --create-check --frequency=10m -- --grep="@critical"
+```
+
+## Configuration
+
+### Artifacts
+
+Configure in your `playwright.config.ts`:
+
+```javascript
+use: {
+  trace: 'on',      // Always capture traces
+  video: 'retain-on-failure',   // Record videos on failure
+  screenshot: 'only-on-failure' // Take screenshots on failure
+}
+```
+
+View all artifacts in Checkly's UI after test runs.
+
+### Environment variables
+
+```bash
+# Pass individual variables and pull all tests tagged with regression into a check suite
+npx checkly pw-test --env ENVIRONMENT_URL="https://staging.acme.com" -- --grep="@regression"
+
+# Load env from file
+npx checkly pw-test --env-file=".env.staging" -- --project=chromium
+```
+
+## Common patterns
+
+### E-commerce monitoring
+```bash
+# Checkout flow
+npx checkly pw-test --create-check --frequency=15m --location=us-east-1 -- --project="Desktop Chrome" --grep="checkout"
+
+# Mobile experience
+npx checkly pw-test --create-check --frequency=30m -- --project="iPhone-13" --grep="@mobile-critical"
+```
+
+## Test sessions
+
+All runs record automatically. You can add custom names:
+
+```bash
+npx checkly pw-test --test-session-name="Pre-deployment check" -- --project=chromium
+```
+
+## Best practices
+
+1. Tag tests clearly: `@smoke`, `@regression`, `@critical`
+2. Enable artifacts in `playwright.config.ts`
+3. Start with critical user paths
+4. Test from multiple regions
+5. Balance monitoring frequency with importance
+
+## Troubleshooting
+
+**Tests fail in Checkly but pass locally**
+- Check local environment variables match the ones in Checkly
+- Verify URLs are accessible from test locations
+- Review failures and traces in Checkly UI
+
+**Missing artifacts**
+- Confirm they're set as `on` in `playwright.config.ts`
```

Comment:
Aha! this answers my question from above but I still think it's worth calling out there. :)